scripts/generate_json_docs.py

```diff
@@ -156,36 +157,38 @@ def from_pdoc(cls, element):
         components = element.refname.split('.')
 
         mod = __import__(components[0])
-
         for comp in components[1:]:
             mod = getattr(mod, comp)
 
-        build_source(mod, method)
+        # Get method line number.
+        method.add_source_line(get_source_line_number(mod))
+
+        # Get method Examples.
+        examples = get_examples_from_docstring(element.docstring)
+        if examples:
+            method.add_example(examples)
 
         if element.docstring:
             if not isinstance(element, pdoc.Class) and element.cls:
-                cls = element.cls.cls
+                clas = element.cls.cls
```

Comment:
I really need a better name for this.
`cls` is passed in and I wanted to preserve that. As I was trying to debug an issue I kept getting things confused.  Maybe I should change it to something like `pdoc_class`?.